### PR TITLE
Renamed log levels and added Xcode project for iOS framework

### PR DESCRIPTION
**Pull request by sundance2000**
Hi,

I really like the QorumLogs and use it now in my projects. The only thing, which bothers me, is the naming of the logging functions. To my opinion, function names should be descriptive like "QLogError" and should not contain just three characters like "QL4". This improves the readability of code and makes it easier to identify the log level by name. Maybe you agree and want to add this to your code.

I also like to make frameworks out of external code, so it can be reused in other projects without copying the source files into the project. I know this can also be done with CocoaPods and you already provide a podspec file. But currently, this system still seems very unstable to me and I want to control code and frameworks by myself. Hence, I also added a minimal Xcode project to provide a framework for iOS. I hope I set the bundle identifier and the project settings correctly. I hope you like the idea of the framework and add it to your repository.

Thank you for your time and your good work.
Kind Regards,
Christian

### CODE REVIEW

**Comment by Esqarrouth** [comment]
Thanks for using QorumLogs!

For the 1st one, naming: One of the main features of this is being autocomplete friendly and being easy to type. People who use it daily remember what the log levels and what color is associated with them. But yes, new people coming into the code might not understand what is going on at first sight. However with a little click they can learn it: http://prntscr.com/aivwqu

Changing the names also means that every active project has to rename all their logs inside their code. We might need to change the name in upcoming Swift updates or rebranding or more people want this feature in the next months. When that happens I'll reconsider this. 

But I urge you to use a little trick I saw a few people use so they don't have to import QorumLogs everywhere. 

Basically what they do is make their own custom interface in their Configs or Utilities file like this:

``` swift
import QorumLogs

func logThisDebug<T>(debug: T, _ file: String = default, _ function: String = default, _ line: Int = default) {
        QL1("my log", file, function, line)
}
```

This way, they can change the name as they want and they don't have to import this in everyfile.

**Comment by Esqarrouth** [comment]
For your second commit, thanks! I've been thinking about adding this myself but was too lazy. I'll merge it but I can't seem to ignore the first commit you made. Have any ideas with that?

**Comment by sundance2000** [comment]
To your first answer:
OK, I see that my suggestion is problematic for projects already using the current function names. Thank you for the example code for custom interfaces. I will use those until another solution is possible.

**Comment by sundance2000** [comment]
To your second answer:
It seems that it is not possible to pick only some of the commits in a pull request. I created a feature branch only containing the second commit and started a new pull request. I hope this works now. Sorry for the confusion.
